azure-pipelines.yml

```diff
@@ -507,37 +507,35 @@ jobs:
     inputs:
       ArtifactName: debian
 
+- job: BuildUbuntuDisco
+  displayName: Build Ubuntu Disco Package
 
-
-# - job: BuildDebianWheezy
-#   displayName: Build Debian Wheezy Package
-
-#   dependsOn: BuildPythonWheel
-#   condition: and(succeeded(), in(variables['Build.Reason'], 'IndividualCI', 'BatchedCI', 'Manual'))
-#   pool:
-#     vmImage: 'ubuntu-16.04'
-#   steps:
-#   - task: DownloadBuildArtifacts@0
-#     displayName: 'Download Build Artifacts'
-#     inputs:
-#       artifactName: pypi
-
-
-#   - task: Bash@3
-#     displayName: 'Build Debian Wheezy'
-#     inputs:
-#       targetType: 'filePath'
-#       filePath: scripts/release/debian/pipeline.sh
-#     env:
-#       DISTRO: wheezy
-#       DISTRO_BASE_IMAGE: debian:wheezy
-
-
-#   - task: PublishBuildArtifacts@1
-#     displayName: 'Publish Artifact: debian'
-#     inputs:
-#       ArtifactName: debian
-
+  dependsOn: BuildPythonWheel
+  # condition: and(succeeded(), in(variables['Build.Reason'], 'IndividualCI', 'BatchedCI', 'Manual'))
+  condition: succeeded()
```

Comment:
What's the rationale for 515 over 514?  Does this mean this build will always run?

Reply:
Just temporary! This is the only change in 70fb79c, and I'll revert it before merging this PR. I just wanted to actually run this code before it gets merged :)

Reply:
[Looks like the build worked](https://dev.azure.com/azure-public/azcli/_build/results?buildId=5412), reverting now.